src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -172,3 +174,28 @@ def validate(self, attrs):
             raise ValidationError({"author": "author required for public integrations"})
 
         return attrs
+
+
+class SentryAppAvatarSerializer(Serializer):
```

Comment:
Moved from `avatar.py` for clarity. It's now just referenced on the endpoint (`sentry_app_avatar.py`) instead.